### ISSUE TITLE
Is the crl any improvement effect?

### ISSUE THREAD

**Issue by AllenMao** [GdMacmillan/kaggle-protein-classification#7]
hi, @GdMacmillan , I used crl and trained with celeba face attributes dataset, but improvement is about 0.3%. So I want to know the improvement effect of your experiments in this repositories. Thanks!

**Comment by GdMacmillan** [comment]
Hi @AllenMao, I appreciate your inquiry. Due to the project timeline, i didn't have much time to use the CRL loss function after I wrote it. And we don't have a k8s cluster up at the moment. A good experiment would be proving an improvement over standard cross entropy loss in a repeatable fashion. I have moved on for the time being but is this something you could do? You can contact the lead researcher who has worked on this at q.dong@qmul.ac.uk.

**Comment by AllenMao** [comment]
@GdMacmillan Thank for your reply, I have sent an email to the author.
